repondeur/zam_repondeur/templates/amendements.html

```diff
@@ -763,6 +770,7 @@ <h3>Supprimer</h3>
         editLink.addEventListener('click', e => {
             const thisURL = new URL(window.location.href)
             const linkURL = new URL(e.target.href)
+            thisURL.hash = ''
```

Comment:
The hash is not included in the value of `back` below, so I don't think this is needed

Reply:
I think there is a tricky case when there is already a remaining hash coming from a previous back/save action which keep it. I'll double check